packages/core/index.d.ts

```diff
@@ -25,7 +25,7 @@ declare const middy: <H extends AsyncHandler<C>, C extends Context = Context>(ha
 
 declare namespace middy {
   interface Middy<T, R, C extends Context = Context> extends Handler<T, R> {
-    use: <M extends MiddlewareObject<T, R, C>>(middleware: M) => Middy<T, R, C>;
+    use: <M extends MiddlewareObject<T, R, C>>(input: M | M[]) => Middy<T, R, C>;
```

Comment:
Very good one! Should we just rename `input` to `middlewares`? I believe it's slightly more descriptive